File: remove_notranslate.js
const fs = require('fs');
const files = ['index.html', 'main.js'];

files.forEach(file => {
    let content = fs.readFileSync(file, 'utf8');
    content = content.replace(/translate="no"/g, '');
    content = content.replace(/notranslate/g, '');
    fs.writeFileSync(file, content, 'utf8');
    console.log(`Removed translation blocks from ${file}`);
});

File: remove_notranslate.py
import os

files = ['index.html', 'main.js']

for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    content = content.replace('translate="no"', '')
    content = content.replace('notranslate', '')
    
    with open(file, 'w', encoding='utf-8') as f:
        f.write(content)
    
    print(f"Removed translation blocks from {file}")
